samples/aspnetcore/SwaggerSample/ImplicitApiVersionParameter.cs

```diff
@@ -0,0 +1,72 @@
+﻿namespace Microsoft.Examples
+{
+    using Microsoft.AspNetCore.Mvc.ApiExplorer;
+    using Swashbuckle.AspNetCore.Swagger;
+    using Swashbuckle.AspNetCore.SwaggerGen;
+    using System.Collections.Generic;
+    using System.Linq;
+
+    /// <summary>
+    /// Represents the Swagger/Swashbuckle operation filter used to document the implicit API version parameter.
+    /// </summary>
+    public class ImplicitApiVersionParameter : IOperationFilter
```

Comment:
Is it used here for query string case only, isn't it?
And for version in path case I have to use SetVersionInPath filter, right?